pkg/broker/rbac.go

```diff
@@ -1,15 +1,24 @@
 package broker
 
 import (
+	"fmt"
+
 	v1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+const (
+	SubmarinerBrokerNamespace = "submariner-k8s-broker"
+	SubmarinerBrokerSA        = "submariner-k8s-broker-client"
+	SubmarinerBrokerRole      = "submariner-k8s-broker-client"
```

Comment:
As above, it would be better if we could avoid exporting these constants.

Reply:
ack, thanks.

Reply:
They’re still exported ;-).